### DISCUSSION TITLE
Which POST_APPLET_* to use after mounting via FSTAB_*

### DISCUSSION THREAD

**Discussion by intelliant01** [ltsp/ltsp#694] [Q&A] [answered]
Am mounting a NFS share via FSTAB_01 in ltsp.conf.
Which POST_APPLET_* may I use to copy over files from the mounted share?
I tried POST_INIT_* but it does not work.

**Comment by alkisg** [comment]
Try POST_SERVICE_x.

**Comment by intelliant01** [comment]
> Try POST_SERVICE_x.

Did not work.

These are my entries in `ltsp.conf` -
```
FSTAB_05="server:/opt/exelearning26 /opt/exelearning26 nfs defaults,nolock,rsize=32768,wsize=32768 0 0"
POST_SERVICE_EXE_MENU="cp /opt/exelearning26/exelib/exe.desktop /usr/share/applications/; cp /opt/exelearning26/exelib/exe.png /usr/share/icons/"
```

**Comment by alkisg** [comment]
First of all, you can use symlinks, e.g.:

```shell
POST_INIT_SYMLINS_EXE="ln -s /opt/exelearning26/exelib/exe.desktop /usr/share/applications/exe.desktop"
```

Second, if you run the `cp` commands you mentioned manually after the client boots, do they work?

**Comment by intelliant01** [comment]
Symlinks work but not copy.

> Second, if you run the `cp` commands you mentioned manually after the client boots, do they work?

Yes, they do when I try via Epoptes's `Open Terminal`->`Root, locally`

With symlinks this is solved for now. But why does `cp` not work?

**Comment by alkisg** [comment, marked as answer]
The mount is probably not up yet by the time `ltsp service` runs. You'd have to create a script instead, and wait until it's mounted. Or create a systemd unit that depends on the mount; but a how-to for that is outside the scope of LTSP support.
